assets/js/main.js

```diff
@@ -387,27 +387,31 @@ $(function(){
 });
 // open accordion for direct links to panel id
 $(window).load(function() {
-  var getHash = location.hash; //get hash from js object location
+  const getHash = location.hash; //get hash from js object location
+
   function activateAccordion(id) {
-    if (id.length)//check if hash isn't empty
-    {
-      var accordion = $('a[href="' + id + '"]');
-      $panelHeight = accordion.parents('.panel').height();
-      console.log('Offset Top: ' + accordion.parents('.panel').offset().top);
-      $('html,body').animate({
-          scrollTop: accordion.parents('.panel').offset().top //scroll to accordion
-      }, 100, function () {
-          accordion.click(); //simulate click
-          accordion.parents('.panel').find('.panel-title a.no-scroll i.fa').addClass('panel-isOpen'); //point caret down for oepned panel
-      });
+    const accordion = $('a[href="' + id + '"]');
+    const $panelHeight = accordion.parents('.panel').height();
```

Comment:
Won't `.height()` blow up if there's nothing that matches `.panel`? It seems like the check you want is `accordion.length > 0`.

Reply:
I don't think that would be the right conditional in this case. `accordion.length > 0` will be true in this case and will still throw an error when it gets to the panel height. I noticed the code that retrieves the parents($('.panel'))  always returns something. Not exactly sure where the panel class is in this case but the `panel` class isn't the actual issue. I tried this $('a[href="' + location.hash + '"]').parents($('.panel'))  in my console for https://gruntwork.io/guides/#kubernetes/ and it returns an output . The error is coming from .height()of the panel returning null for some pages that's why I'm using that condition

Reply:
Check both `accordion.length > 0` and `accordion.parents('.panel').length > 0`.

Reply:
That's true. Not sure why I never thought of that